abuse-policy.md

```diff
@@ -0,0 +1,18 @@
+Title: Abuse Policy
+
+When IO Cooperative receives an abuse report indicating that a member's IP
+address is the origin of abuse, for example a report of attempts to gain
+unauthorized access to a system originating from a member's IP address, IO 
+Cooperative will notify the member that they have 48 hours to respond to the
+report, illustrating how the abuse has been stopped or how the report was 
+inaccurate. If the member has not responded or resolved the abuse within that
+time their IP address will be blocked from inbound and outbound network access.
+
+If IO Cooperative deems that the reported abuse is affecting service 
+availability for other cooperative members, the IP address will be immediately
+blocked and the member will be notified of the abuse and the block.
+
+Once the abuse is resolved the member's IP address may be unblocked. IO 
+Cooperative will terminate membership of any member who either repeatedly allows
```

Comment:
Add a link to the bylaws section that describes the process for terminating a member: https://iocoop.org/policies/bylaws#section-503-expulsion

Reply:
Will do